java/memory/memory-netty/src/main/java/io/netty/buffer/UnsafeDirectLittleEndian.java

```diff
@@ -31,16 +31,7 @@
  * Netty classes and underlying Netty memory management.
  */
 public class UnsafeDirectLittleEndian extends WrappedByteBuf {
-
-  public static final boolean ASSERT_ENABLED;
   private static final AtomicLong ID_GENERATOR = new AtomicLong(0);
-
```

Comment:
Is this just cleanup? trying to understand it's relation with JIRA

Reply:
These were unused, I was examining all static initializer blocks so I just decided to remove these